api/default_env.tres

```diff
@@ -0,0 +1,7 @@
+[gd_resource type="Environment" load_steps=2 format=2]
```

Comment:
Are these files only needed for testing? If so, we should move them to the `tests/` folder.

Reply:
Yup, that sounds like a good idea. I double check the [default environment]'s doc (https://docs.godotengine.org/en/stable/tutorials/3d/environment_and_post_processing.html#default-environment), and seems like it is only used for 3d scenes. Since the GUI we are using is a 2d scene this file can be removed from the project.
I also refactored the location of the gui scene and moved it into the ```test/``` folder.